fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -438,6 +440,16 @@ public static String analyzeType(Map<String, String> properties) {
         return type;
     }
 
+    public static boolean analyzeColocateMV(Map<String, String> properties) throws AnalysisException {
```

Comment:
How about reuse method “ boolean analyzeBooleanProp(Map<String, String> properties, String propKey, boolean defaultVal) ”  instead of write a new one？